src/stories/elements/e-checkbox.stories.ts

```diff
@@ -12,8 +13,9 @@ export default {
 };
 
 // More on component templates: https://storybook.js.org/docs/vue/writing-stories/introduction#using-args
-const Template = {
-  render: (args) => ({
+const Template: StoryObj = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  render: (args: any) => ({
```

Comment:
note: I don't like having any here. But I think for now it is sufficient enough.

Reply:
I couldn't get it to work without it. Maybe I missed something here.